components/textfield/metadata/textfield.yml

```
@@ -1,6 +1,6 @@
 name: Text field
 status: Verified
-description: A single-line text field (aka textfield)
+description: A single-line text field using the `<input>` element with a `type="text"`.
```

```suggestion
description: A single-line text field using the `<input>` element`.
```
Is this only limited to `type="text"`? I think it can also be `url`,`email`, or `password`.